load-test-framework/src/main/java/com/google/pubsub/flic/controllers/Client.java

```diff
@@ -81,6 +81,8 @@ static String getTopicSuffix(ClientType clientType) {
       case CPS_GCLOUD_PUBLISHER:
       case CPS_GCLOUD_SUBSCRIBER:
         return "gcloud";
+      case CPS_GCLOUD_PYTHON_PUBLISHER:
+        return "python";
```

Comment:
This doesn't look right. Until now, these strings reflected the service used for publishing, not the language of the client library. Or are they supposed to reflect the client library type?